docs/about-mina/faq.mdx

```diff
@@ -50,8 +46,11 @@ Mina's consensus mechanism is an implementation of Ouroboros proof of stake. Due
 
 Yes, check out these block explorers:
 
-- https://minascan.io
-- https://minataur.net 
+- [Mina Explorer](https://minaexplorer.com/)
```

Comment:
Let's remove Mina Explorer if it won't support mainnet after the HF.

Reply:
removed minaexplorer